Xamarin.Essentials/TextToSpeech/TextToSpeech.shared.cs

```diff
@@ -63,7 +63,7 @@ internal static float PlatformNormalize(float min, float max, float percent)
         }
     }
 
-    public partial struct Locale
+    public class Locale
```

Comment:
Rationale for changing this?

Reply:
All containing properties are reference types. The struct is not immutable. There is (afaik) no reason to have this as a value type.